virttest/nfs.py

```diff
@@ -196,7 +196,7 @@ def __init__(self, params):
             params.get("setup_remote_nfs") == "yes"
             or params.get("setup_local_nfs") == "yes"
         ):
-            if "Ubuntu" in distro_details or "rhel" in distro_details:
+            if "Ubuntu" in distro_details or "rhel" in distro_details or "fedora" in distro_details:
```

Comment:
```suggestion
            if distro_details.upper() in ("UBUNTU", "RHEL", "FEDORA", ):
```
After I've checked the usage about return of `distro.detect().name`, I personally suggest to use `tuple` for comparing between strings.

Reply:
@nickzhq , thanks a lot for your time to review this PR! I totally agree with you, will update the PR in same manner.